cmd/ops_agent_uap_plugin/service_linux.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !windows
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os/exec"
+	"path"
+	"regexp"
+	"strings"
+	"syscall"
+
+	"google.golang.org/grpc/status"
+
+	pb "github.com/GoogleCloudPlatform/ops-agent/cmd/ops_agent_uap_plugin/google_guest_agent/plugin"
+)
+
+const (
+	OpsAgentConfigLocationLinux = "/etc/google-cloud-ops-agent/config.yaml"
```

Comment:
When does the config get copied to this location?

If it happens at install-time, i.e. when the guest agent extracts the tarball, then it's going to overwrite a user's existing config if they have previously installed the Ops Agent via the native package manager.

Why aren't we storing the config alongside the binary?